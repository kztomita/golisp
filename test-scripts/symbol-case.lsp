(print 'foo)
(print 'FOO)
(setq FOO 1)
(print foo)
(print FOO)

