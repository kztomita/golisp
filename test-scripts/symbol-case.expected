
foo 
foo 
1 
1 
